util.groovy

```diff
@@ -0,0 +1,502 @@
+@GrabResolver(name = 'jcenter', root = 'http://jcenter.bintray.com/')
+@Grab(group = 'org.ajoberstar', module = 'grgit', version = '1.9.3')
+import org.ajoberstar.grgit.Grgit
+import org.ajoberstar.grgit.Remote
+import groovy.json.JsonSlurper
+
+//Load gradlew properties to check for github settings.
+Properties properties = new Properties()
+new File("gradle.properties").withInputStream {
+    properties.load(it)
+}
+
+/**
+ * Default settings for modules.
+ * githubRepo stores the Repository name.
+ * targetDirectory stores the working directory.
+ * displayName is used to generalize the script.
+ */
+githubRepo="Terasology"
+targetDirectory="modules"
+displayName="Module"
+//Main Logic for changing githubRepo, targetDirectory & displayName
```

Comment:
add blank line before this, and make this a Javadoc-style comment (to maintain consistency throughout)

Reply:
Okay!